Tensile/Source/client/source/HardwareMonitor.cpp

```diff
@@ -326,7 +326,11 @@ namespace Tensile
                 v = 0;
             for(auto& v : m_fanValues)
                 v = 0;
-
+                
+            m_freqValues.clear();
+            m_powerValues.clear();
+            m_temperatureValues.clear();
```

Comment:
is there a reason why we're not pre-setting the size and clearing them like the other things we log, rather than emptying the vector and pushing data in later?

Reply:
we need all individual data later for detailed logging so keeping all the values. Earlier metrics it accumulates (adds) all the individual data. separating out and logging the individual data is overhead.